allensdk/brain_observatory/stimulus_info.py

```diff
@@ -446,7 +465,15 @@ def pixels_to_visual_degrees(self, n, distance_from_monitor, small_angle_approxi
         else:
             return 2*np.arctan(n*1./2*self.pixel_size / distance_from_monitor) * 57.2958  # radians to degrees
 
-    def lsn_image_to_screen(self, img, stimulus_type, origin='lower', background_color=STIMULUS_GRAY):
+    def visual_degrees_to_pixels(self, vd, distance_from_monitor, small_angle_approximation=True):
+
+        if small_angle_approximation == True:
+            return vd*(distance_from_monitor/self.pixel_size/57.2958)
```

Comment:
This conversion constant is used in a couple places, perhaps it could be defined once?